ansible/vars/enikshay/enikshay_public.yml

```diff
@@ -0,0 +1,287 @@
+SITE_HOST: 'enikshay.in'
+#CAS_SITE_HOST: 'cas.commcarehq.org'
+#ENIKSHAY_SITE_HOST: 'enikshay.commcarehq.org'
```

Comment:
It doesn't have http forwarding so you'll need to use https://enikshay.commcarehq.org. looks like it also doesn't have static files.

I'm pretty sure you need to keep that variable `enikshay.commcarehq.org` so that phones can continue submitting to it after the migration to ctrl-s. @proteusvacuum @ctsims will probably know more about the migration strategy, but I'd be surprised if that's not the chase